### PR TITLE
58 old tutorials new db

### LINKED ISSUE #58: Update old tutorials to reflect new database

### PR DESCRIPTION
**Pull request by rdahlman3**
Updates the old tutorials and code snippets to reflect the updated database in PR #32 in the replservice repo.

Closes #58

### CODE REVIEW

File: content/efcore/getting-started/loading-related-data.md [hunk 1]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var book = context.Book 
-		.Include(b => b.Editions) 
-			.ThenInclude(e => e.Publisher)
-		.Single(b => b.BookId == 1) ; 
+		.Include(b => b.CheckoutRecords) 
+			.ThenInclude(cr => cr.Reader)
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: I understand why you did `cr`, but the standard would be to just do `c`

File: content/efcore/getting-started/loading-related-data.md [hunk 2]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var author = context.Authors 
 		.Include(a => a.Books) 
-			.ThenInclude(b => b.Editions) 
-				.ThenInclude(e => e.Publisher)
+			.ThenInclude(b => b.CheckoutRecords) 
+				.ThenInclude(cr => cr.Reader)
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: `c`

File: content/efcore/getting-started/loading-related-data.md [hunk 3]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
-	var book = context.Books 
-		.Include(b => b.Editions) 
-			.ThenInclude(e => e.Publisher) 
-		.Include(b => b.Author)		
-		.Single(b => b.Title.Contains("Call of the Wild")); 
+	var book = context.Books
+        .Include(cr => cr.CheckoutRecords)
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: `c`

File: content/efcore/getting-started/loading-related-data.md [hunk 4]
```diff
@@ -154,18 +154,18 @@ using (var context = new LibraryContext())
...
     var book = context.Books
-        .Single(b => b.Title == "The Scarlet Plague"); 
+        .Single(b => b.Title.Contains("Orient Express")); 
  
     var year = context.Entry(book) 
-        .Collection(b => b.Editions) 
+        .Collection(b => b.CheckoutRecords) 
         .Query() 
-        .Min(e => e.Year); 
+        .Min(cr => cr.CheckoutDate);
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: `c`

File: content/efcore/getting-started/loading-related-data.md [hunk 5]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
+		.Include(b => b.CheckoutRecords) 
+			.ThenInclude(cr => cr.Reader)
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=then-include-method} 
 :::
  
-We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the publisher information for the different editions of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, edition, and publisher information. 
+We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the reader information for the checkout records of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, checkout records, and reader information.
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: For example, if we want to find out all of the readers who have checked out books written by Frederick Douglass, we will need to chain `ThenIncludes` to get the associated books, checkout records, and reader information.

File: content/efcore/getting-started/loading-related-data.md [hunk 6]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=multiple-include-methods} 
 :::
  
 ## Multi-layer Inclusion 
  
-Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the publisher information for the editions of a book like so: 
+Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the reader information for the checkout records of a book like so:
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: For example, we can determine all of the readers who have checked out a specific book by using `Include` to load the book's checkout records and then using `ThenInclude` to load the reader associated with each checkout record as shown below:

File: content/efcore/getting-started/loading-related-data.md [hunk 7]
```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
...
+			.ThenInclude(b => b.CheckoutRecords) 
+				.ThenInclude(cr => cr.Reader)
 		.Single(a => a.LastName == "Douglass"); 
 } 
 ``` 
 :::repl{data-name=then-include-method-chain} 
 :::
  
-A mix of `Include` and `ThenInclude` commands can also be chained together to include related data from multiple layers across related entities. In the following example, we are looking for the author, editions, and edition publishers of _The Call of the Wild_. 
+A mix of `Include` and `ThenInclude` commands can also be chained together to include related data from multiple layers across related entities. In the following example, we are looking for the author, checkout records, and the readers of those records of _Murder on the Orient Express_.
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: In the following example, we chain together `Include` and `ThenInclude` commands to load the author of _Murder on the Orient Express_, as well as all of the readers who have checked it out.

File: content/efcore/getting-started/samples/loading-related-data/include-and-then-include-method-chain/Program.cs [hunk 8]
```diff
@@ -9,16 +9,16 @@ public static void Main()
         using (var context = new LibraryContext())
         {
             var book = context.Books
-                .Include(b => b.Editions)
-                    .ThenInclude(e => e.Publisher)
+                .Include(b => b.CheckoutRecords)
+                    .ThenInclude(cr => cr.Reader)
```
Comment (helmoski) [inline content/efcore/getting-started/samples/loading-related-data/include-and-then-include-method-chain/Program.cs]: `c`

File: content/efcore/getting-started/samples/loading-related-data/min-method/Program.cs [hunk 9]
```diff
@@ -9,14 +9,14 @@ public static void Main()
         using (var context = new LibraryContext())
         {
             var book = context.Books
-                .Single(b => b.Title == "The Scarlet Plague");
+                .Single(b => b.Title.Contains("Orient Express")); 
+ 
+            var earliestCheckout = context.Entry(book) 
+                .Collection(b => b.CheckoutRecords) 
+                .Query() 
+                .Min(cr => cr.CheckoutDate);
```
Comment (helmoski) [inline content/efcore/getting-started/samples/loading-related-data/min-method/Program.cs]: `c`

File: content/efcore/getting-started/samples/loading-related-data/then-include-method-chain/Program.cs [hunk 10]
```diff
@@ -8,17 +8,17 @@ public static void Main()
...
         using (var context = new LibraryContext())
         {
-            var author = context.Authors
-                .Include(a => a.Books)
-                    .ThenInclude(b => b.Editions)
-                        .ThenInclude(e => e.Publisher)
+            var author = context.Authors 
+                .Include(a => a.Books) 
+			        .ThenInclude(b => b.CheckoutRecords) 
+				        .ThenInclude(cr => cr.Reader)
```
Comment (helmoski) [inline content/efcore/getting-started/samples/loading-related-data/then-include-method-chain/Program.cs]: `c`

File: content/efcore/getting-started/samples/loading-related-data/then-include-method/Program.cs [hunk 11]
```diff
@@ -9,13 +9,13 @@ public static void Main()
         using (var context = new LibraryContext())
         {
             var book = context.Books
-                .Include(b => b.Editions)
-                    .ThenInclude(e => e.Publisher)
-                .Single(b => b.BookId == 1);
+                .Include(b => b.CheckoutRecords) 
+                    .ThenInclude(cr => cr.Reader)
```
Comment (helmoski) [inline content/efcore/getting-started/samples/loading-related-data/then-include-method/Program.cs]: `c`

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/loading-related-data.md (hunk 5 above)
Comment (rdahlman3) [reply content/efcore/getting-started/loading-related-data.md]: My b on all of these wordings - I kinda just replaced the class names without registering how it sounded, and I definitely could have done better

Comment (helmoski) [reply content/efcore/getting-started/loading-related-data.md]: No problem. It's also easier to edit the wordings after seeing what you already have, so it didn't hurt anything.

File: content/efcore/getting-started/loading-related-data.md [hunk 12]
```diff
@@ -154,18 +154,18 @@ using (var context = new LibraryContext())
 :::repl{data-name=count-method} 
 :::
  
-Likewise, we can find the year of the oldest edition of a book using the `Min` method: 
+Likewise, we can find the date of the first checkout record of a book using the `Min` method:
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: ~~...date of the first time a specific book was checked out using...~~

Actually, could we switch this to "Likewise, we can find the most recent date a specific book was checked out using the `Max` method:"

File: content/efcore/getting-started/tutorial-model-reference.md [hunk 13]
```diff
@@ -2,54 +2,75 @@
  
 In code, our entity classes look like:
```
Comment (helmoski) [inline content/efcore/getting-started/tutorial-model-reference.md]: Here is what the entity classes that make up our example data model look like:

File: content/efcore/getting-started/manipulating-entities.md [hunk 14]
```diff
@@ -100,7 +100,7 @@ using (var context = new LibraryContext())
 { 
     var book = new Book() 
     { 
-        BookId = 1, 
+        Id = 1, 
         Title = "Frankenstein: or, The Modern Prometheus", 
         Genre = "Science Fiction", 
         PublicationYear = 1818
```
Comment (helmoski) [inline content/efcore/getting-started/manipulating-entities.md:106]: Line 132 talks about editions and publishers still

File: content/efcore/getting-started/samples/shared/LibraryContext.cs [hunk 15]
```diff
@@ -3,10 +3,11 @@
 
 public class LibraryContext : DbContext
```
Comment (helmoski) [inline content/efcore/getting-started/samples/shared/LibraryContext.cs:4]: There is another LibraryContext.cs that needs to be updated as well:
`content\efcore\getting-started\samples\manipulating-entities\shared\LibraryContext.cs`

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/loading-related-data.md (hunk 12 above)
Comment (rdahlman3) [reply content/efcore/getting-started/loading-related-data.md]: I like that better, makes more sense as an example

File: content/efcore/getting-started/manipulating-entities.md (hunk 14 above)
Comment (rdahlman3) [reply content/efcore/getting-started/manipulating-entities.md:106]: balls

File: content/efcore/getting-started/samples/shared/Address.cs [hunk 16]
```diff
@@ -0,0 +1,6 @@
+public class Address
+{
+    public int Id { get; set; }
+    public String City { get; set; }
```
Comment (helmoski) [inline content/efcore/getting-started/samples/shared/Address.cs]: In order for the examples to work, we either need to add `using System;` or make the data type of lines 4 and 5 `string`.

**Review by helmoski — CHANGES_REQUESTED** [review]

**Review by helmoski — APPROVED** [review]

**Pull request merged**